src/pybind/mgr/volumes/fs/operations/versions/subvolume_base.py

```diff
@@ -227,3 +227,25 @@ def create_base_dir(self, mode):
             self.fs.mkdirs(self.base_path, mode)
         except cephfs.Error as e:
             raise VolumeException(-e.args[0], e.args[1])
+
+    def info (self):
+        subvolpath = self.metadata_mgr.get_global_option('path')
+        etype = self.metadata_mgr.get_global_option(MetadataManager.GLOBAL_META_KEY_TYPE)
+        st = self.fs.statx(subvolpath, cephfs.CEPH_STATX_BTIME |
+                                            cephfs.CEPH_STATX_SIZE,
+                                            cephfs.AT_SYMLINK_NOFOLLOW)
+        usedbytes = st["size"]
+        try:
+            nsize = int(self.fs.getxattr(subvolpath, 'ceph.quota.max_bytes').decode('utf-8'))
+        except cephfs.NoData:
+            nsize = 0
+
+        try:
+            data_pool = self.fs.getxattr(subvolpath, 'ceph.dir.layout.pool').decode('utf-8')
+        except cephfs.Error as e:
+            raise VolumeException(-e.args[0], e.args[1])
+
+        return {'path': subvolpath, 'type': etype,
+            'bytes_quota': "no-limit" if nsize == 0 else nsize, 'bytes_used': int(usedbytes),
+            'bytes_pcent': "no-limit" if nsize == 0 else '{0:.2f}'.format((float(usedbytes) / nsize) * 100.0),
```

Comment:
Better if this matches the output of  **resize**
See, https://github.com/ceph/ceph/blob/octopus/src/pybind/mgr/volumes/fs/volume.py#L191
i.e., 'bytes_percent' is 'undefined' if nsize == 0 and
       'bytest_quota' is same 'nsize'

Reply:
I think the outputs of our APIs need to be consistent.  I prefer if it matches the 'resize' output results. Setting the **bytest_pcent** to *undefined* should be OK. 

You're right, setting **bytest_quota** to zero is misleading. 
For the output of **resize** command,  **bytes_quota** is set to 'infinite'. Does that work for you?

Reply:
Yes, that works for me. I will change 'bytes_pcent' and 'bytes_quota' to 'undefined' and 'infinite' respectively.

Reply:
Done